src/components/doc-editor/doc-editor-app.tsx

```diff
@@ -29,6 +31,10 @@ const {document: documentURL, readOnly, noStorage, unwrapped } = urlParams;
 const savedDocString = noStorage ? undefined : window.sessionStorage.getItem(kDocEditorDocKey);
 const initialDoc = savedDocString ? JSON.parse(savedDocString) : defaultDocumentModel;
 
+// gate the AI summary features with query parameters
+const showAiSummary = new URLSearchParams(window.location.search).get("showAiSummary") === "true";
+const includeModelInAiSummary = new URLSearchParams(window.location.search).get("includeModelInAiSummary") === "true";
+
```

Comment:
It'd probably be better to add these to the `urlParams` global which is already being imported. And also update the Readme which provides documentation for each url param.